lib/hammer_cli_foreman_virt_who_configure/config.rb

```diff
@@ -117,7 +117,7 @@ class FetchCommand < HammerCLIForeman::Command
 
       failure_message _('Could not fetch the Virt Who configuration')
 
-      option ['--output', '-o'], 'FILE', _('File where the script will be written.')
+      option ['-o', '--output'], 'FILE', _('File where the script will be written.')
```

Comment:
Option description must not end with a dot. It's here for a while and I think it's time to fix it as well.

Reply:
Thanks, I removed the dot.